C-Plus-Plus/math/Smallest_divisible_number.cpp

```diff
@@ -0,0 +1,43 @@
+//Given a number n find the smallest number evenly divisible by each number 1 to n.
+
+#include<bits/stdc++.h> 
+using namespace std; 
+
+// Function returns the gcd of first n numbers 
+long long gcd(long long a,long long b ){
+
+
+    if (a == 0)
+        return b;
+    return gcd(b % a, a);
+}
+// Function returns the lcm of first n numbers 
+long long lcm(long long n) 
+{ 
+	long long ans = 1;	 
+	for (long long i = 1; i <= n; i++) 
+		ans = (ans * i)/(gcd(ans, i)); 
+	return ans; 
+} 
+
+// Driver program to test the above function 
+int main() 
+{ 
+	long long n;
+	cout<<"Enter the number:";
+    cin>>n;             //Taking input
+	cout << lcm(n); 
+	return 0; 
+} 
+
+/*
+##Example :
+
+Input : n = 4
```

Comment:
Edit the sample input/output exactly how it will look in the terminal when the program is run.

Reply:
Done